templates/user_form2.tt.html

```diff
@@ -0,0 +1,241 @@
+<script>
+
+    function normalize_string_value(inputfield) {
+
+    var value = inputfield.value.toLowerCase();
+
+    value = value.replace(new RegExp(" ", 'g'),"-");
+    value = value.replace(new RegExp("[àáâãäå]", 'g'),"a");
+    value = value.replace(new RegExp("æ", 'g'),"ae");
+    value = value.replace(new RegExp("ç", 'g'),"c");
+    value = value.replace(new RegExp("[èéêë]", 'g'),"e");
+    value = value.replace(new RegExp("[ìíîï]", 'g'),"i");
+    value = value.replace(new RegExp("ñ", 'g'),"n");
+    value = value.replace(new RegExp("[òóôõö]", 'g'),"o");
+    value = value.replace(new RegExp("œ", 'g'),"oe");
+    value = value.replace(new RegExp("[ùúûü]", 'g'),"u");
+    value = value.replace(new RegExp("[ýÿ]", 'g'),"y");
+    value = value.replace(new RegExp("[^a-zA-Z0-9-]", 'g'),"-");
+    value = value.replace(new RegExp("-+", 'g'),"-");
+    value = value.replace(new RegExp("^-"),"");
+
+    inputfield.value = value;
+    }
+</script>
+
+[% IF action == 'display' || action == 'none' %]
+[% INCLUDE 'error_list.tt.html' %]
+[% END %]
+
+[% IF action == 'display' %]
+
+    <!-- Start form -->
+
+    <form method="post" action="/cgi/user2.pl" enctype="multipart/form-data">
+		[% FOREACH section IN sections %]
+			[% IF user OR section.id == "user" %]
+				[% FOREACH field IN section.fields %]
+					[% IF field.field == 'userid' %]
+                        [% IF type == 'edit' %]
+						    <label for="[% field.field %]"> [% lang(field.label2) %]</label>
+                            <p> [% user_id_field %] </p>
+                        [% ELSE %]
+                            <label for="[% field.field %]">  [% lang(field.label2) %] [% lang('username_info')  %]</label>
+                            <input type="text" id="[% field.field %]" name="[% field.field %]" value="[% field.value %]" autocomplete="[% field.field %]" onkeyup="normalize_string_value(this)" />
+                        [% END %]
+					[% ELSIF field.type == 'text' %]
+                        <label for="[% field.field %]">[% field.label %]</label>
+                        <input type="text" id="[% field.field %]" name="[% field.field %]" value="[% field.value %]" autocomplete="[% field.field %]"/>
+					[% ELSIF field.type == 'password' %]
+						<label for="[% field.field %]">[% lang(field.label2) %]</label>
+						<input type="password" id="[% field.field %]" name="[% field.field %]" value="[% user_password_field %]" autocomplete="new-password"/>
```

Comment:
` "\n<tr><td>$Lang{password}{$lang}</td><td>"
	. password_field(-name=>'password', -value=>$user_ref->{password}, -autocomplete=>'new-password', -override=>1) . "</td></tr>"
	. "\n<tr><td>$Lang{password_confirm}{$lang}</td><td>"
	. password_field(-name=>'confirm_password', -value=>$user_ref->{password}, -autocomplete=>'new-password', -override=>1) . "</td></tr>"`

This is the code snippet of `Users.pm`, here for both `password` and `confirm_password` value is same. That's why I stored it in one variable and used it for both fields. 

But I guess it can be replaced.